### PR TITLE
update tiller deployment spec

### PR DESCRIPTION
**Pull request by wlan0**
@alena1108

### CODE REVIEW

**Comment by alena1108** [comment]
@wlan0 could you test to launch a couple of apps from the templates before we tag the release?

File: package/addons-update.sh [hunk 1]
```diff
@@ -6,6 +6,15 @@ while ! kubectl --kubeconfig=/etc/kubernetes/ssl/kubeconfig --namespace=kube-sys
   sleep 2
 done
 
+cat > /tmp/rancher-service-account.yaml << EOF
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+      name: "io-rancher-system"
```
Comment (alena1108) [inline package/addons-update.sh:13]: can it be io.rancher.system? are dots allowed?

**Comment by alena1108** [comment]
Should we also update Dashboard with the same logic? I'd prefer it to be a part of kubernetes-package rather than rancher fork of k8s

File: package/addons-update.sh (hunk 1 above)
Comment (wlan0) [reply package/addons-update.sh:13]: dots are not allowed in the service account token names.

**Pull request merged**